src/components/ContentWidgets/Separator/Separator.css

```diff
@@ -0,0 +1,56 @@
+.separator {
```

Comment:
Combine some of your css selectors by using commas. For example you can merge together something like this:

```css
.width-range::-webkit-slider-thumb, .width-range::-moz-range-thumb {
  background: black;
}
```

Reply:
It seems like combining vendor-specific pseudo-elements will confuse the browser and ignore the css block. The slider handle will disappear like this:
<img width="480" alt="Screen Shot 2019-07-14 at 9 10 54 PM" src="https://user-images.githubusercontent.com/29179779/61195418-b4bdbd80-a67c-11e9-87d0-75b1da6a45f4.png">

Reply:
Interesting. No problem then, feel free to ignore